.editorconfig

```diff
@@ -13,5 +13,4 @@ insert_final_newline = true
 trim_trailing_whitespace = false
 
 [{package,bower}.json]
-indent_style = space
```

Comment:
Does this mean that indents are now tabs, or indents are spaces by default?

Reply:
It was redundant as we define to use spaces for all type of files:
```
[*]
indent_style = space
```